index.html

```diff
@@ -1,369 +1,257 @@
-          </div>
-          <div class="title-desktop">
-            <h1>I’m Alexandra <br>
-              Glad to see you!</h1>
-          </div>
-          <div class="titletext">
-            <p>
-              I’m a software developer! I can help you build a product , feature
-              or website Look through some of my work and experience! If you
-              like what you see and have a project you need coded, don’t
-              hestiate to contact me.
-            </p>
-          </div>
-
-          <div class="connect-about">
-            <h2>LET'S CONNECT</h2>
-          </div>
-          <div id="footer-icon" class="footer-icons-about">
-            <ul>
-              <li>
-                <a href="#footer-icon">
-                  <img src="./images/twitter (2).svg" alt="twitter logo" />
-                </a>
-              </li>
-              <li>
-                <a href="#footer-icon">
-                  <img src="./images/Linkedin icon.svg" alt="linkedin" />
-                </a>
-              </li>
-              <li>
-                <a href="#footer-icon"
-                  ><img src="./images/m-icon.svg" alt="m logo"
-                /></a>
-              </li>
-              <li>
-                <a href="#footer-icon"
-                  ><img src="./images/github-icon.svg" alt="github logo"
-                /></a>
-              </li>
-              <li>
-                <a href="#footer-icon"
-                  ><img src="./images/hand-peace-icon.svg" alt="peace logo"
-                /></a>
-              </li>
-            </ul>
-          </div>
+    <!-- ===============MODAL WINDOW=============   -->
+    <article class="modal" id="modal">
+      <div class="modal-header">
+        <h2 class="title"></h2>
+        <div class="subtitle">
+          <h3>CANOPY</h3>
+          <img  src="https://res.cloudinary.com/dapnjt4ah/image/upload/v1675866221/Counter_f68nkh.svg" alt="Counter icon" />
+          <h3 class="light-text">Back End Dev</h3>
+          <img  src="https://res.cloudinary.com/dapnjt4ah/image/upload/v1675866221/Counter_f68nkh.svg" alt="Counter icon" />
+          <h3 class="light-text">2015</h3>
         </div>
-      </section>
-    
-      <!-- =====TONIC-PAGE-==== -->
-      <section class=" container">
-        <div class="tonic1 tonic-container">
-          <div class="image-tonic">
-            <img class="img-tanne" src="./images/tanne.svg" alt="mobile tanne image" />
-            <img class="img-tanneDSKT" src="./images/SnapshootPortfolio.svg" alt="desktop tanne image" />
-          </div>
+        <button data-close-button class="close-btn" type="button">
+          &times;
+        </button>
+      </div>
+      <div class="modal-body">
+        <img src="" class="modal-img" alt="" />
+        <div class="modal-desc">
+          <p></p>
+          <ul class="modal-categories"></ul>
+        </div>
+      </div>
+      <div class="modal-buttons">
```

Comment:
- [ ] You did well 😊, but the previous reviewer meant the padding of the li to increase it not to increase the padding at the bottom of the ul, so please try to increase the padding of the li of the language to be like the Figma Design👌

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/217634587-81e69765-8a0f-44b3-9825-f82ebdd60953.png)   | ![image](https://user-images.githubusercontent.com/83360636/217636082-40e0275c-66ab-4e7e-8f66-ee100d632d7b.png)  |